<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="animalEditor">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Component</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="chooser">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="del">
           <property name="text">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="modify">
           <property name="text">
            <string>Modify</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Size &amp; Color</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QDoubleSpinBox" name="size">
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Texture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="color">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="view">
         <property name="text">
          <string>View</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bend">
         <property name="text">
          <string>Bend</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="move">
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rotate">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="scale">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="info">
         <property name="text">
          <string>Component: </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Q3dsScene" name="scene" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QToolButton" name="addFrame">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeFrame">
           <property name="text">
            <string> -</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveStepUp_2">
           <property name="text">
            <string>^</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveStepDown_2">
           <property name="text">
            <string>v</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="actionEditor">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Action</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="actions">
           <item>
            <property name="text">
             <string>Freeze</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addKeyFrame_2">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeKeyFrame_2">
           <property name="text">
            <string> -</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Key Frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="keyframes"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QToolButton" name="addKeyFrame">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeKeyFrame">
           <property name="text">
            <string> -</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveKeyFrameStepUp">
           <property name="text">
            <string>^</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveKeyFrameStepDown">
           <property name="text">
            <string>v</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Delay</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="delay">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="curve">
           <item>
            <property name="text">
             <string>Flat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Drop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Raise</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hill</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grove</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="actionPreviewForward">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="actionPreviewSpeed"/>
         </item>
         <item>
          <widget class="QToolButton" name="actionPreviewBack">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionResize"/>
    <addaction name="actionRotate"/>
    <addaction name="actionMove"/>
    <addaction name="actionBend"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCancel_Edit"/>
    <addaction name="actionSave_Edit_Frame"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPan"/>
    <addaction name="actionSpin_Around"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="separator"/>
    <addaction name="actionTop"/>
    <addaction name="actionFront"/>
    <addaction name="actionLeft"/>
    <addaction name="actionRight"/>
    <addaction name="action45_Degree"/>
    <addaction name="action135_Degree"/>
    <addaction name="separator"/>
    <addaction name="actionDoll_sitemap"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Frame"/>
    <addaction name="actionRemove_Frame"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
   </widget>
   <widget class="QMenu" name="menuFrame">
    <property name="title">
     <string>Frame</string>
    </property>
    <addaction name="actionNew_from_Current"/>
    <addaction name="actionNew_from_Frame_0"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHow_to"/>
    <addaction name="actionOnline_manual"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModel"/>
   <addaction name="menuView"/>
   <addaction name="menuFrame"/>
   <addaction name="menuAction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTop"/>
   <addaction name="actionFront"/>
   <addaction name="actionLeft"/>
   <addaction name="actionRight"/>
   <addaction name="action45_Degree"/>
   <addaction name="action135_Degree"/>
   <addaction name="separator"/>
   <addaction name="actionSpin_Around"/>
   <addaction name="actionPan"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="separator"/>
   <addaction name="actionCancel_Edit"/>
   <addaction name="actionSave_Edit_Frame"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionBend">
   <property name="text">
    <string>Bend</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionSpin_Around">
   <property name="text">
    <string>Spin</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionFront">
   <property name="text">
    <string>Front</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="action45_Degree">
   <property name="text">
    <string>45</string>
   </property>
  </action>
  <action name="action135_Degree">
   <property name="text">
    <string>135</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Figure</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Figure</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>New Action</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove Action</string>
   </property>
  </action>
  <action name="actionAdd_2">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove_2">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionAdd_Frame">
   <property name="text">
    <string>Add Frame</string>
   </property>
  </action>
  <action name="actionRemove_Frame">
   <property name="text">
    <string>Remove Frame</string>
   </property>
  </action>
  <action name="actionAnimal_Editing_Mode">
   <property name="text">
    <string>Animal Editing Mode</string>
   </property>
  </action>
  <action name="actionAction_Editing_Mode">
   <property name="text">
    <string>Action Editing Mode</string>
   </property>
  </action>
  <action name="actionDoll_sitemap">
   <property name="text">
    <string>Figure sitemap</string>
   </property>
  </action>
  <action name="actionNew_from_Current">
   <property name="text">
    <string>New from Current</string>
   </property>
  </action>
  <action name="actionNew_from_Frame_0">
   <property name="text">
    <string>New from Frame 0</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHow_to">
   <property name="text">
    <string>How to</string>
   </property>
  </action>
  <action name="actionOnline_manual">
   <property name="text">
    <string>Online manual</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCancel_Edit">
   <property name="text">
    <string>Cancel Edit Frame</string>
   </property>
  </action>
  <action name="actionSave_Edit_Frame">
   <property name="text">
    <string>Save Edit Frame</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3dsScene</class>
   <extends>QWidget</extends>
   <header>q3dsscene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
